### PR TITLE
Lizards can now be bought from cargo

### PR DESCRIPTION
**Pull request by Hardly3D**
## About The Pull Request
This PR adds a lizard crate to cargo, containing a _friendly_ lizard (As long as you don't attack them :^) ).
Lfzy79Fa6W

dreamseeker_ch6YX7y9pW

## Why It's Good For The Game
Adds another way to get lizard pets aside from stealing the one in janitoral's closet or by bugging a xenobiologist into spamming critters.

## Changelog

:cl: Hardly
add: You can now buy lizards through cargo.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -2239,6 +2239,14 @@
 					/mob/living/simple_animal/hostile/retaliate/snake)
 	crate_name = "snake crate"
 
+/datum/supply_pack/critter/lizard
+	name = "Lizard Crate"
+	desc = "Hisss! Containssss a friendly lizard, not to be confusssed with a lizardperssson."
+	cost = CARGO_CRATE_VALUE * 4
+	access_view = ACCESS_JANITOR
```
Comment (Time-Green) [inline code/modules/cargo/packs.dm:2246]: I don't think this has to be janitor only right?

**Comment by Shadowflame909** [comment]
the power cargo now possesses is Lizardriffic

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Hardly3D) [reply code/modules/cargo/packs.dm:2246]: This was added in order to keep it consistent with the other pet crates also having access_view (not to be confused with access which locks crates to set access) set to specific jobs (eg. goat to cook, fox to captain, parrots to CE), hence why it's set to janitor. This only affects the IRN tablet program and can be seen by anyone on the cargo supply console/request console.

Comment (Time-Green) [reply code/modules/cargo/packs.dm:2246]: Ah okay, I misunderstood then. Seems good!

**Comment by notfrying1pans** [comment]
trafficking

**Comment by PowerfulBacon** [comment]
Should have a 5% chance to spawn a lizard person

**Pull request merged**